miro_preprocessor/xml_to_json_converter.Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM python:3-alpine3.6
+LABEL maintainer "Alex Chan <a.chan@wellcome.ac.uk>"
+LABEL description "Copy records from Miro XML dumps into DynamoDB"
+
+RUN apk update
+
+# Required for lxml
+RUN apk add build-base libxml2-dev libxslt-dev python3 python3-dev
+
+COPY xml_to_json_converter/requirements.txt /requirements.txt
+RUN pip3 install -r requirements.txt
+
+COPY xml_to_json_converter /app
+
+ENTRYPOINT ["/app/run.py"]
```

Comment:
Minor: I’d love us to get out of the habit of giving these scripts such generic names.

`run_xml_to_json_converter.py`?

Reply:
This is part of a larger app contained in this folder (and is not the only part).

Its name denotes its part of the app.